apps/api/scripts/calculate-next-season-eligibility.ts

```diff
@@ -136,6 +163,8 @@ Examples:
         and(
           // Only claims with duration > 0 (actual stakes)
           gte(convictionClaims.duration, 1n),
+          // Only stakes from claims before the reference time
+          lte(convictionClaims.blockTimestamp, referenceTime),
```

Comment:
Realized this needs to be added to be sure that stakes that happened after the ref time aren't included.